### ISSUE TITLE
Batch size selection

### ISSUE THREAD

**Issue by guilherme-coelho-isi** [JonasSchult/Mask3D#131]
Hello, thank you for the excellent work!
I'm trying to train on smaller batches, so I can maintain the same resolution but still have the same results on a weaker GPU.
Is there any parameter to change the batch size?
Thank you in advance, Jonas.

Salutations,
Guilherme Coelho Fermino.

**Comment by guilherme-coelho-isi** [comment]
Found it! However, now I'm a bit worried about the learning rate, which its implementation on Mask3D is inverselly proportional to the batch size, and I couldn't find a parameter to change that. Shouldn't it be direcly proportional do the batch size? (as I've only used softgroup, maybe I could be mistaken with the implementation of Mask3D)

**Comment by bh-cai** [comment]
Dear sir,I have the same issue, can you tell me where to change the batch_size?  Thank you very much!

**Comment by guilherme-coelho-isi** [comment]
The parameter inputed is `data.batch_size`, then you input the value you want or your machine can handle. Like:

`python main_instance_segmentation.py \`
`  data.batch_size=1`

**Comment by 0nandon** [comment]
@guilherme-coelho-isi  Hi! did u address the learning rate issue above? And did you try the training with 1 batch size? How is the performance compared to the original Mask3D?
